src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,253 @@
+    annotation_offset : int, optional (default: 8.5)
+        The offset of the annotations from the base labels.
+    annotation_font : dict, optional (default: {'size': 'smaller'})
+        The *Matplotlib* compatible font of the annotations.
+    bin_path : str, optional
+        Path of the RNAplot binary.
+    """
+
+    # Check if RNAplot is installed
+    if shutil.which(bin_path) is None:
+        raise FileNotFoundError(
+            'RNAplot is not installed at the specified location, unable to '
+            'plot secondary structure.'
+        )
+
+    # Get the unknotted base pairs
+    if pseudoknot_order is None:
+        pseudoknot_order = pseudoknots(base_pairs, max_pseudoknot_order=0)[0]
+    unknotted_base_pairs = base_pairs[pseudoknot_order == 0]
+
+    # If `bond_linewidth` is not an array, extrapolate
+    if not isinstance(bond_linewidth, np.ndarray):
+        bond_linewidth = np.full(base_pairs.shape[0], bond_linewidth)
+
+    # If `bond_color` is not an array, extrapolate
+    if not isinstance(bond_color, np.ndarray):
+        bond_color = np.full(base_pairs.shape[0], bond_color)
+
+    # If `background_color` is not an array, extrapolate
+    if not isinstance(base_box, np.ndarray):
+        base_box = np.full(length, base_box)
+
+    # By default pseudoknotted bonds are denoted as dashed lines, while
+    # unknotted bonds are denoted as solid lines
+    if bond_linestyle is None:
+        bond_linestyle = np.full(base_pairs.shape[0], 'solid', dtype='object')
+        bond_linestyle[pseudoknot_order != 0] = 'dashed'
+    # If `bond_linestyle` is not an array, extrapolate
+    elif not isinstance(bond_linestyle, np.ndarray):
+        bond_linestyle = np.full(base_pairs.shape[0], bond_linestyle)
+
+    # If pseudoknots are not to be drawn, remove pseudoknotted bonds, 
+    # regardless of the given linestyles
+    if not draw_pseudoknots:
+        bond_linestyle[pseudoknot_order != 0] = 'None'
+    
+    
+    # If no specific annotation positions are given, annotate every
+    # second base pair
+    if annotation_positions is None:
+        annotation_positions = range(0, length, 2)
+
+    # Get coordinates for secondary structure plot
+    coordinates = RNAplotApp.compute_coordinates(
+    base_pairs=unknotted_base_pairs, 
+    length=length,
+    bin_path=bin_path,
+    layout_type=layout_type
+    )
+
+    # Rotate Coordinates
+    if angle != 0:
+        angle = np.deg2rad(angle)
+        R = np.array(
+            [[np.cos(angle), -np.sin(angle)],
+            [np.sin(angle),  np.cos(angle)]]
+        )
+        for i, coord in enumerate(coordinates):
+            coordinates[i] = np.dot(R, coord)
+
+    # Create Plot
+    fig, ax = plt.subplots(figsize=(10, 10))
+
+    # Remove axes and frame
+    ax.set_xticks([])
+    ax.set_yticks([])
+    plt.box(False)
+
+    # Define buffer area (Border)
+    coord_range = abs(np.max(coordinates)) + abs(np.min(coordinates))
+    buffer = 0.03*coord_range
```

Comment:
`0.03` seems very specific. Please comment and create a parameter or constant for this value.